src/Microsoft.ML.OnnxTransformer/OnnxUtils.cs

```diff
@@ -202,9 +195,15 @@ public OnnxVariableInfo(string name, OnnxShape shape, Type typeInOnnxRuntime, Da
             {
                 // Load ONNX model file and parse its input and output schema. The reason of doing so is that ONNXRuntime
                 // doesn't expose full type information via its C# APIs.
-                ModelFile = modelFile;
                 var model = new OnnxCSharpToProtoWrapper.ModelProto();
-                using (var modelStream = File.OpenRead(modelFile))
+                // If we own the model file set the DeleteOnClose flag so it is always deleted.
+                if (ownModelFile)
```

Comment:
>if [](http://example.com/codeflow?start=16&length=2)

nit: please use parentheses with the if statements.

Reply:
This is the style that most of ML.NET, and this file, use for doing single line if statements. To keep things consistent I am going to leave it as is.